go.mod

```
@@ -45,7 +45,7 @@ require (
 	github.com/filecoin-project/go-cbor-util v0.0.1
 	github.com/filecoin-project/go-commp-utils/v2 v2.1.0
 	github.com/filecoin-project/go-crypto v0.1.0
-	github.com/filecoin-project/go-f3 v0.8.6
+	github.com/filecoin-project/go-f3 v0.8.8-0.20250707135223-3b1d3fcfb899
```

```suggestion
	github.com/filecoin-project/go-f3 v0.8.8-0.20250707135223-3b1d3fcfb899 // dependency-check-ignore: unknown
```

you'll need explicitly acknowledge you're being naughty to get by the dependency checker in CI